OtripleS.Portal.Web.Tests.Unit/Services/Courses/CourseServiceTests.Exceptions.RetrieveAll.cs

```diff
@@ -0,0 +1,130 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE AS LONG AS SOFTWARE FUNDS ARE DONATED TO THE POOR
+// ---------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Moq;
+using OtripleS.Portal.Web.Models.Courses;
+using OtripleS.Portal.Web.Models.Courses.Exceptions;
+using Xunit;
+
+namespace OtripleS.Portal.Web.Tests.Unit.Services.Courses
+{
+    public partial class CourseServiceTests
+    {
+        [Theory]
+        [MemberData(nameof(CriticalApiExceptions))]
+        public async Task ShouldThrowCriticalDependencyExceptionOnRetrieveAllIfCriticalDependencyExceptionOccursAndLogItAsync(
```

Comment:
You can't have any line over 120 characters.
Try to trim down the list name as `ShouldThrowCriticalDependencyOnRetrieveAllIfCriticialErrorOccursAndLogItAsync`

Reply:
@hassanhabib thanks for noticing, fixed